### ISSUE TITLE
[Web Version of Zoom Meeting] Audio not working with uBlock Enabled

### ISSUE THREAD

**Issue by cqfllp** [uBlockOrigin/uAssets#7854]
### URL(s) where the issue occurs

`https://zoom.us/(any_meeting_link)`

### Describe the issue

With uBlock extension enabled and nothing else, even if the filter is not filtering anything, Zoom web meeting will have no sound. I can't hear anyone speaking but they can hear me if i speak with microphone enabled. The zoom meeting is completely silent.

### Versions

- Browser/version: Google Chrome Version 85.0.4183.83 (Official Build) (64-bit)
- uBlock Origin version: 1.29.2

### Settings

Default Settings are able to reproduce the error/my custom settings will get the error as well of course.
-

### Notes

Both Firefox and incognito chrome works as intended(without uBlock Origin). Whenever I enable uBlock Origin and re-enter the Zoom meeting, no sound can be heard even if I put its domain into the trusted site list or disable uBlock Origin using its button. I have to emphasize that no filters are applied. uBlock Origin **extension** has to be DISABLED for the audio to work as normal.

I have not modified anything. The last month I can confirm there were no issues related to the audio in Zoom was around May 2020.

**Comment by krystian3w** [comment]
Zoom had problems with the stability of the service yesterday.

Unfortunately, it looks as if you are accusing the JavaScript code  of the June-September 2020 addon version of being flawed and spoiling web applications, because there are no by filters, e.g. EasyPrivacy, that damage the application or the application still no works after deactivating filtering for one domain.

For the test you can deactivate all filters and see if the additive code interacts with the application directly.

**Comment by cqfllp** [comment]
> Zoom had problems with the stability of the service yesterday.
> 
> Unfortunately, it looks as if you are accusing the JavaScript code of the June-September 2020 addon version of being flawed and spoiling web applications, because there are no by filters, e.g. EasyPrivacy, that damage the application or the application still no works after deactivating filtering for one domain.
> 
> For the test you can deactivate all filters and see if the additive code interacts with the application directly.

My friend hasn't updated his chrome version for a long time and his browser works. Looks like it's guaranteed to get this error if we have a chrome version that is pushed later after May. 

I don't think it has anything to do with the stability of the service because this issue already persists for like a week(school just started so i found it)

**Comment by llacb47** [comment]
@cqfllp does disabling the setting `Prevent WebRTC from leaking local IP addresses` under the settings tab make a difference?

**Comment by krystian3w** [comment]
Default disabled, for manual enable.

**Comment by cqfllp** [comment]
> @cqfllp does disabling the setting `Prevent WebRTC from leaking local IP addresses` under the settings tab make a difference?

It is ... weird. Yesterday I used all methods to test it(including disabling this setting). But as you reminded me about it, I just tested again and it did make a difference. Thank you!

But another thing is that this setting was enabled all the time. I have been using this extension for years and there were no issues. Assuming I just started using Zoom in March, it didn't affect the audio back the time. Is there any way I can disable this setting specifically for this website but not all of them?
